tasks/service-wrapper/app/service/service-runner.ts

```diff
@@ -0,0 +1,164 @@
+import { PythonShell } from 'python-shell';
+import { Response } from 'express';
+import { readdirSync } from 'fs';
+import { spawn } from 'child_process';
+import env from '../util/env';
+import log from '../util/log';
+import sem from '../util/semaphore';
+import WorkItem from '../../../../app/models/work-item';
+
+export interface ServiceResponse {
+  batchCatalogs?: string[];
+  error?: string;
+}
+
+/**
+ * Runs a service request
+ *
+ * @param operation - The requested operation
+ * @param res - The Response to use to reply
+ */
+export function runServiceForRequest(operation, res: Response): void {
+  const options = {
+    pythonOptions: ['-u'], // get print results in real-time
+    cwd: '/home',
+    args: [
+      `${env.harmonyService}`,
```

Comment:
I'm confused what `env.harmonyService` is at this point? It looks like it would be something like `harmonyservices/netcdf-to-zarr:latest`? I don't understand how it's configurable to tell the service wrapper what the python module name is that needs to be run to start the service. Further, what if you don't run the service code as a module (`-m` option) but instead using a run-script (like l2ss-py does) 
https://github.com/podaac/l2ss-py/blob/develop/pyproject.toml#L55
https://github.com/podaac/l2ss-py/blob/develop/docker/Dockerfile#L56

Reply:
Yes, using env.harmonyService for this was incorrect. This code has been replaced in any case.